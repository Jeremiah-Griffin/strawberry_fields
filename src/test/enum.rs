use strawberry_fields::strawberry_fields;

#[strawberry_fields(u32)]
enum Dummy {
    First,
    Second,
}
fn main() {}
