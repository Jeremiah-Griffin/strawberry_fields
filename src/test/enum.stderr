error: custom attribute panicked
 --> src/test/enum.rs:3:1
  |
3 | #[strawberry_fields(u32)]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = help: message: StrawberryFields may only be derived for structs.: Error("expected `struct`")
